server/serverAPI/model/user/responseSchema/IUserResponse.ts

```diff
@@ -0,0 +1,5 @@
+import IBaseUser from "../IBaseUser";
+import IContactInformation from "../IContactInformation";
+import IIdentification from "../IIdentification";
+
+export default interface IUserResponse extends IBaseUser, IIdentification, IContactInformation {}
```

Comment:
Newline.

Also, if this interface doesn't specify any behavior, why do we have it?

Reply:
So... because UserResponse is consisted of specific fields.... Like I don't simply return User object to the user, only parts that I want to return... That's why I user UserResponse to combine all parts of the user that I want to return.